content/guides/analog-discovery/_index.md

```diff
@@ -0,0 +1,886 @@
+
+
+### Running The Full Test Suite
+
+We'll walk through every important step, below, and that is a good way to get to know the Analog Discovery.  However, if you want to just get going and explore things on your own, you may simply wire things up and run the pre-built workspace on the AD3, you may just follow the directions in this section.
+
+## What you need
+
+   1. An Analog Discovery 3 and its *WaveForms* program installed, connected to the computer;
+   2. The *tt05analogdemoFull.dwf3work* workspace opened in *WaveForms*;
+   3. A TT05 demoboard and ASIC, also connected to the computer via USB;
+   4. To have things wired up
+
+The basic wiring is:
+
+```
+AD3     Demoboard Input
+1+      m33 (top header)
+1-      GND (top header, but really anywhere on demoboard)
+2+      m35 (top header)
+2-      GND
+0       in0
+1       in1
+2       in2
+3       in3
+9       in5
+10      in6
+GND     GND
+```
+
+If you also want to run the oscillator frequency vs core voltage characterization, you'll need to
+
+   * depopulate F2 from the demoboard
+   * wire *V+* from the AD3 to the 1V8 pin on the demoboard top header
+
+
+This is optional as you may skip the test at the prompt
+{{< figure src="images/47.png" title="Test run prompt" >}}
+
+or simply get the boring result of no-frequency-variation (since the core voltage won't actually be set by the AD3).
+
+## How to run
+
+
+Assuming you've wired everything up, these are the steps to run the test suite.  Following the entire sequence is only important if you've set things to drive the core supply from the AD3, but it doesn't hurt even if you haven't
+
+   1. connect the AD3 to USB
+   2. open the WaveForms program and load the workspace file
+   3. switch to the *Supplies* tab, ensure Positive supply is set to 1.8V and press the "play" button on that tab to enable the core voltage.  Make sure both the "master enable" and "positive supply" have green checkmarks;
+   4. switch to the *Script* tab, find the *powerupHighZ* tab in there and press its "play" button to run it, then click *Yes* to the "Do you want to put all DIO into High-Z" prompt
+   5. connect the demoboard to USB
+   6. press "play" button on the *demoboardsetup* script tab, and specify the correct "com port" (ttyACM0 is the default, and usually valid, under linux)
+   7. press "play" on the *powerupHighZ*, again... but this time click "No" to the first prompt and then "Yes" to "Do you wish to restore Patterns signals to PP?"
+
+Now, everything should be in a good state, and you shouldn't have to go through that dance again unless you power down.
+
+Clicking the "play" button on the *tests* tab in Script will run the tests--simply follow the prompts.
+
+{{< figure src="images/49.png" title="DAC plot prompt" >}}
+
+
+
```

Comment:
add a heading here like 'full walkthrough'. then that can also be linked to above when mentioning it.